feature/security/gnsi/certz/tests/client_certificates/README.md

```diff
@@ -0,0 +1,133 @@
+# CERTZ-1: gNSI Client Certificate Tests
```

Comment:
I might have asked this earlier, but why is it that this looks like the original file almost entirely?